index.js

```diff
@@ -9,6 +9,32 @@ var name = require('./package.json').name;
 
 var VERSION_REGEX = /(\w+)-(.+)/;
 
+/**
```

Comment:
This comment appears inaccurate; if not present, it seems to be a rejected promise.